packages/reports/src/Components/market-symbol-icon-row.tsx

```diff
@@ -4,14 +4,14 @@ import { Icon, Popover, IconTradeTypes } from '@deriv/components';
 import { getMarketName, getTradeTypeName } from '../Helpers/market-underlying';
 
 type TMarketSymbolIconRow = {
-    icon: string;
+    icon?: string | null;
```

Comment:
A question: if the icon is an optional value. Are there any cases where the `null` passed and can we refactors those calls instead?

Reply:
None seems that was a mistake I made from before. I have updated it in the new commit. 👍🏼